lib/author.rb

```diff
@@ -0,0 +1,27 @@
+class Author
+    attr_reader :name, :books
+
+    def initialize(description)
+        @name = description[:first_name] + " " + description[:last_name]
```

Comment:
Would like to see this as a method that does this, the less instance variables the better.